cypress/integration/T2.js

```diff
@@ -0,0 +1,118 @@
+import LoginPage from '../objects/LoginPage';
+import TimeLoggin from '../objects/TimeLoggin';
+import Common from '../objects/Common';
+const common = new Common();
+const timeLoggin = new TimeLoggin();
+const loginPage = new LoginPage();
+
+describe('Sourcebooks login', function() {
+    it('Should display validation for empty user after attempted loggin', function () {
+        
+        loginPage.visit();
+        loginPage.getUserValidationIndicator().should('not.visible')
+        loginPage.getSubmitButton().click();
+        loginPage.getUserValidationIndicator().should('be.visible')
+    })
+
+    it('Should be able to login with user Jaunius_Mykolaitis to role Admin', function () {
```

Comment:
why there is a '_' between your name? it is not used like that in actual test case :)